.lycheeignore

```diff
@@ -0,0 +1,9 @@
+file:///github/workspace/\w+/\$%7Bhref%7D
+file:///github/workspace/.+/_media/
+file:///github/workspace/marblerun/.+\.md
```

Comment:
_media must be ignored because these links are relative to the respective root dir and are only correctly resolved in docsify.
marblerun mds are ignored for now because marblerun doc also uses relative paths to the root dir, whereas the other docs use real relative paths. Only the latter can be correctly resolved by the link checker.